utils.go

```diff
@@ -442,31 +442,14 @@ func cleanupWorkDirs() error {
 		}
 	}
 
-	// Remove the sysbox lib dir's content.
-	if _, err := os.Stat(sysboxLibDir); err == nil {
-		if err := removeDirContents(sysboxLibDir); err != nil {
-			return err
-		}
-	} else if !os.IsNotExist(err) {
-		return err
+	// Remove the sysbox lib dir
```

Comment:
Have you checked this works in our test-setup? sysboxLibDir is a bind-mount within the priv container so don't think you can delete the folder itself ...

Reply:
Correct ... I had to make a change in the test container for this to work. PR coming soon.